app/src/main/java/com/devshish/internship/presentation/ui/service/client/MediaBrowserClient.kt

```diff
@@ -39,6 +42,15 @@ class MediaBrowserClient(context: Context) {
         )
     }
 
+    fun getSong(): Song = mediaController.metadata.toSong()
+
+    fun playSong(song: Song) {
+        val extras = Bundle().apply {
```

Comment:
Lets use kotlin magic and write extension val for bundle (I will help you with this)